src/main/java/org/opensearch/forecast/settings/ForecastEnabledSetting.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package org.opensearch.forecast.settings;
+
+import static java.util.Collections.unmodifiableMap;
+import static org.opensearch.common.settings.Setting.Property.Dynamic;
+import static org.opensearch.common.settings.Setting.Property.NodeScope;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.opensearch.common.settings.Setting;
+import org.opensearch.timeseries.settings.DynamicNumericSetting;
+
+public class ForecastEnabledSetting extends DynamicNumericSetting {
+
+    /**
+     * Singleton instance
+     */
+    private static ForecastEnabledSetting INSTANCE;
+
+    /**
+     * Settings name
+     */
+    public static final String FORECAST_ENABLED = "plugins.forecast.enabled";
+
+    public static final String FORECAST_BREAKER_ENABLED = "plugins.forecast.breaker.enabled";
+
+    public static final String FORECAST_DOOR_KEEPER_IN_CACHE_ENABLED = "plugins.forecast.door_keeper_in_cache.enabled";;
```

Comment:
Will we also have a setting to enable/disable interpolation or we wont be doing it at all for forecasting?

Reply:
We won't do it at all for forecasting. It is a legacy for AD only.